apps/web/pages/teams/index.tsx

```diff
@@ -28,10 +27,10 @@ function Teams() {
   );
 }
 
-export const getServerSideProps = async (context: GetServerSidePropsContext) => {
+export const getStaticProps = async () => {
   return {
     props: {
-      ...(await serverSideTranslations(context.locale || "en", ["common"])),
+      ...(await serverSideTranslations("en", ["common"])),
```

Comment:
@Udit-takkar  sorry, missed this comment! This page is static instead of serverside, Peer didn't want to make it serverside because of performance. If I'm correct in that case you can not get the locale from the context, right? Since we don't have the locale as part of our url. So we default to English here, which was okay for Peer.